### DISCUSSION TITLE
Query based on number of foreign relations

### DISCUSSION THREAD

**Discussion by gurbaaz27** [prisma/prisma#21095] [Q&A]
### Question

I have a User Table and Comment Table with a one to many relationship, i.e., a user can have many comments, and a comment will have specific one user. I want to fetch all those users who have atleast 5 comments. For now, what I am doing is fetching all users along with their comments and then iterating through them and checking if `user.comments.length >= 5`. Is there any way to apply this filter within the prisma query itself? I could not find anything relevant on the internet regarding the same.

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @gurbaaz27 👋 

Thank you for raising this question.

Prisma does not yet support filtering in the format you specified. The current approach you're using, fetching all users along with their comments and then filtering the results in your application code, is a valid method. However, I understand that this might not be the most efficient way, especially when dealing with large amounts of data.

Can you please create a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) and document your use case so that our engineers can take a look?

**Reply by ludralph** [reply]
Hey @gurbaaz27,

Do you have any more questions around this issue? If no, it would be really helpful if you could quickly **mark this Discussion as _answered_** to indicate that your issue has been resolved. 

In case you do have any outstanding questions, feel free to let us know and we'd be happy to help!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
